cvmfs/upload_s3.cc

```diff
@@ -398,4 +398,7 @@ bool S3Uploader::PlaceBootstrappingShortcut(const shash::Any &object) const {
   return false;  // TODO(rmeusel): implement
 }
 
+
+int64_t S3Uploader::DoGetObjectSize(const std::string &file_name) { return 0; }
```

Comment:
Same as above.  We can add a TODO here: a `HEAD` request gives us the byte count.